x/ecocredit/basket/utils_test.go

```diff
@@ -0,0 +1,45 @@
+package basket
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFormatBasketDenom(t *testing.T) {
```

Comment:
Moved from `msg_create_test.go` to `utils_test.go` without changing the test cases.